lib/VerifiableCredentialStore.js

```diff
@@ -167,7 +167,15 @@ export default class VerifiableCredentialStore {
 
       // build query to find all VCs that match any combination of type+issuer
       const query = [];
-      const issuers = trustedIssuers.map(({id}) => id);
+      const issuers = trustedIssuers.map(({id}) => {
+        if(!id) {
+          const error =
+            new Error('trustedIssuer without an "id" is unsupported.');
```

Comment:
Break on parenthesis when possible not around equals assignment.

```suggestion
          const error = new Error(
            'trustedIssuer without an "id" is unsupported.');
```

Reply:
Fixed 41a1f07